app.js

```diff
@@ -21,6 +24,22 @@ app.use((err, req, res, next) => {
   }
 });
 
+app.use((err, req, res, next) => {
+  if (err.code === "23502") {
+    res.status(400).send({ msg: "Invalid Post Request" });
+  } else {
+    next(err);
+  }
+});
+
+app.use((err, req, res, next) => {
+  if (err.code === "23503") {
+    res.status(404).send({ msg: "Article Not Found!" });
+  } else {
+    next(err);
+  }
+});
+
```

Comment:
As you're getting a lot of these now, it might be worth considering extracting them out into their own file 😄